Userland/Libraries/LibGUI/Event.h

```diff
@@ -102,6 +103,14 @@ class WMSuperKeyPressedEvent : public WMEvent {
     }
 };
 
```

Comment:
The commit message refers to `Taskbar.app` and `Assistant.app` but `.app` suffixes are not a thing on SerenityOS. :^)

Reply:
Fixed!